File: solutions/1.two-sum/solution.js
function twoSum(nums, target ){
    
    let solution=[];
    let base=0;

    while(base < nums.length){

        for(let i=0; i < nums.length; i++){
                if(nums[base] + nums[i] == target && base != i){ 
                    
                    solution.push(base, i) 
                    return solution
                } 
        }
        base++;
    }
}

let solution=twoSum( [-1,-2,-3,-4,-5], -8)

console.log(solution)

File: solutions/1.two-sum/solution.py
def two_sum(nums, target):
 
    base=0
    while base < len(nums):
        for index in range(len(nums)):
            if nums[base] + nums[index] == target and base != index:
                solution=[base,index]
                return solution
        base +=1


solution=two_sum([-1,-2,-3,-4,-5], -8)

print(solution)